.github/workflows/conventional-pr.yml

```diff
@@ -0,0 +1,43 @@
+name: "Conventional PR"
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  pull_request:
+    branches:
+      - main
+     types: [assigned, opened, synchronize, reopened, edited]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Please look up the latest version from
+      # https://github.com/amannn/action-semantic-pull-request/releases
+      - uses: actions/checkout@v2
+      - uses: amannn/action-semantic-pull-request@v3.4.0
+        with:
+          #Types from: https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines
+          types: |
```

Comment:
As you described scoped in the README it might be great to add them here as well. Our scopes are: api, panel, website, ui, core (changes in the root directory)